api/tests/test_application_api.py

```diff
@@ -323,3 +298,186 @@ def test_application_review_invalid(
     }
     response = user_api_client.post(reverse("application-list"), data, format="json")
     assert response.status_code == 400
+
+
+@pytest.mark.django_db
+def test_application_api_permissions(
```

Comment:
i'd maybe split these into multiple tests instead of asserting so many things in one test, but that can be refactored later.

Reply:
Yeah, I thought about that as valid option but then again I thought it wouldn't be nice either if in same file as other tests there would be 20 permissions tests at the end. We could of course make new test files for each endpoints permissions tests or something. 

I agree that its not beautiful as is either.

Reply:
I agree with splitting to smaller (multiple) tests. Easier to read / make changes if needed etc.

Reply:
Changed to separate tests